lib/krb5/krbhst.c

```diff
@@ -988,6 +988,12 @@ krb5_krbhst_init_flags(krb5_context context,
 					     "tcp", 749));
 	service = "admin";
 	break;
+    case KRB5_KRBHST_READONLY_ADMIN:
```

Comment:
Needs more work.  If you look for `admin_get_next()`, that's what ultimately we'll be using, or want to be using, but it won't be using the right SRV RR label, and it won't be using the right krb5.conf parameter name.